### PR TITLE
Fix validation for range checks

### PR DESCRIPTION
**Pull request by ianco**
Signed-off-by: Ian Costanzo <ian@anon-solutions.ca>

### CODE REVIEW

File: aries_cloudagent/indy/verifier.py [hunk 1]
```diff
@@ -255,17 +255,22 @@ async def pre_verify(self, pres_req: dict, pres: dict):
         for (uuid, req_pred) in pres_req["requested_predicates"].items():
             try:
                 canon_attr = canon(req_pred["name"])
+                matched = False
+                found = False
                 for ge_proof in pres["proof"]["proofs"][
                     pres["requested_proof"]["predicates"][uuid]["sub_proof_index"]
                 ]["primary_proof"]["ge_proofs"]:
+                    print("ge_proof:", ge_proof)
```
Comment (TimoGlastra) [inline aries_cloudagent/indy/verifier.py:263]:
```suggestion
```

**Review by TimoGlastra — APPROVED** [review]

File: aries_cloudagent/indy/verifier.py [hunk 2]
```diff
@@ -255,17 +255,21 @@ async def pre_verify(self, pres_req: dict, pres: dict):
...
                     pred = ge_proof["predicate"]
                     if pred["attr_name"] == canon_attr:
-                        if pred["value"] != req_pred["p_value"]:
-                            raise ValueError(
-                                f"Predicate value != p_value: {pred['attr_name']}"
-                            )
-                        break
-                else:
+                        found = True
+                        if pred["value"] == req_pred["p_value"]:
```
Comment (andrewwhitehead) [inline aries_cloudagent/indy/verifier.py:266]: Does this need to take the type of the values into account or is everything a string?

Comment (ianco) [reply aries_cloudagent/indy/verifier.py:266]: They are predicates so all the values are int's

**Review by swcurran — APPROVED** [review]
LGTM

**Pull request merged**